hack/go-mod-prepare.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+ROOTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+cat > "${ROOTDIR}/go.mod" << EOF
+module github.com/moby/moby
```

Comment:
I realize this is just to satisfy go mod; but just noticed this is using moby/moby; should it be docker/docker? Or no difference whatsoever?

Reply:
I don't think it matters, it all works fine. When I started working on this I didn't realise all import paths were still docker/docker ;)

Reply:
I just noticed that the other `go.mod` file in this script is written with docker/docker, so I've made these two consistent  :)